### PR TITLE
Devops/home page PRA-95

### PR DESCRIPTION
**Pull request by SvetlanaBeynik**
## One Line Description
Home page

## Requirements

_What this code should do_
Render home page

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

## UI/UX Outcome

_Attach a screenshot here_
image

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Review by AnnaPestova1 — APPROVED** [review]

File: src/components/Home.tsx [hunk 1]
```diff
@@ -1,21 +1,70 @@
...
+  CardBody,
+  Flex
+} from "@chakra-ui/react";
 import { Link as RouterLink } from "react-router-dom";
+import { FaFacebookF, FaTwitter } from "react-icons/fa";
+import { BsInstagram } from "react-icons/bs";
+const params = {
+  iconSize: "30",
+  iconSpacing: "10px"
+};
```
Comment (rebekahcallkacz) [inline src/components/Home.tsx:23]: BLOCKING: Is this constant actually used anywhere?

**Review by rebekahcallkacz — CHANGES_REQUESTED** [review]
Congrats on your first PR 🎉 

I ran this locally and the page looks great! My only blocking comment is that I think you have an unused constant.

File: src/components/Home.tsx [hunk 2]
```diff
@@ -1,21 +1,70 @@
...
+  Image,
+  Link,
+  Grid,
+  GridItem,
+  Center,
+  Text,
+  Card,
+  CardBody,
+  Flex
+} from "@chakra-ui/react";
```
Comment (rebekahcallkacz) [inline src/components/Home.tsx:16]: You also should remove any of these imports that you aren't actually using (including the icons below which you aren't displaying anywhere).

File: src/components/Home.tsx (hunk 1 above)
Comment (SvetlanaBeynik) [reply src/components/Home.tsx:23]: Right now it's not used anywhere but it was intended for social media icons

File: src/components/Home.tsx (hunk 2 above)
Comment (SvetlanaBeynik) [reply src/components/Home.tsx:10]: Updated

**Review by rebekahcallkacz — APPROVED** [review]
LGTM!

**Pull request merged**